src/components/text-input/_text-input.scss

```diff
@@ -90,3 +90,88 @@
     }
   }
 }
+
+@mixin text-input--experimental {
+  .#{$prefix}--text-input {
+    @include reset;
+    @include typescale('zeta');
+    background-color: $field-01;
+    width: 100%;
+    height: rem(40px);
+    padding: 0 $spacing-xl 0 $spacing-md;
+    color: $text-02;
+    border: none;
+    outline: 1px solid $ui-04;
+    outline-offset: -1px;
+  }
+
+  .#{$prefix}--text-input ~ .#{$prefix}--label {
+    @include typescale('omega');
+    padding: 0;
+    font-weight: 400;
+    display: block;
+    padding: $spacing-sm 0 $spacing-3xl 0;
+    margin: 0;
+  }
+
+  .#{$prefix}--text-input:required ~ .#{$prefix}--label::after {
+    content: ' *';
+  }
+
+  .#{$prefix}--text-input::-webkit-input-placeholder {
+    color: $text-03;
+  }
+
+  .#{$prefix}--text-input:hover {
+    color: $text-01;
+    outline: 1px solid $brand-01;
+    outline-offset: -1px;
+  }
+
+  .#{$prefix}--text-input:focus,
+  .#{$prefix}--text-input:active {
+    outline: 2px solid $brand-01;
+    outline-offset: -2px;
+  }
+
+  .#{$prefix}--text-input:disabled {
+    cursor: not-allowed;
+    pointer-events: none;
+    color: $text-03;
+  }
+
+  .#{$prefix}--text-input:disabled ~ .#{$prefix}--label {
+    pointer-events: none;
+  }
+
+  .#{$prefix}--text-input[data-invalid],
+  .#{$prefix}--text-input:invalid {
+    outline: 2px solid $support-01;
+    outline-offset: -2px;
+    box-shadow: none;
+    background: $field-01
+      url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32" width="16" height="16" class="ibm-icons ibm-icons--error--filled"><path fill="#da1e28" d="M6.95 26.66A14 14 0 0 0 26.67 6.93zM16 2A14 14 0 0 0 5.35 25.07L25.08 5.34A13.93 13.93 0 0 0 16 2z"/></svg>');
```

Comment:
@IBM/carbon-developers  thoughts on this for the error icon to avoid html changes for now?

Reply:
We may have to convert to Data URI for IE11 support